frontend/server/src/Controllers/Contest.php

```diff
@@ -4877,13 +4878,14 @@ public static function apiContestants(\OmegaUp\Request $r): array {
     }
 
     /**
-     * Archives a contest if user is the creator
+     * Archives or Unarchives a contest if user is the creator
      *
      * @throws \OmegaUp\Exceptions\ForbiddenAccessException
      *
      * @return array{status: string}
      *
      * @omegaup-request-param string $contest_alias
+     * @omegaup-request-param boolean $archive
```

Comment:
```suggestion
     * @omegaup-request-param boolean|null $archive
```

no se por qué aquí no te pidió que lo hicieras nulable...

Reply:
Listo